modules/end-user-guide/pages/mounting-volumes.adoc

```diff
@@ -0,0 +1,66 @@
+:navtitle: Mounting Volumes
+:keywords: user-guide, configuring, user, volumes, persistent, volume, claim, mounting, mount
+:page-aliases:
+
+[id="mounting-volumes_{context}"]
+= Mounting Volumes
+
+You can mount a Persistent Volume Claim (PVC) to your {devworkspace} containers. A PVC is used to request persistent storage resources which are abstracted from the user. This section describes how to automatically mount an existing PVC to all {devworkspace} containers.
+
+.Prerequisites
+
+* A running instance of {prod-short}.
+* You have CLI or GUI access to the {orch-name} cluster of your organization's {prod-short} instance.
+* For CLI users: The `{orch-cli}` command line tool is installed in the operating system you are using.
+
+.Procedure
+
+. Determine an existing PVC in your user {orch-namespace} to mount to all {devworkspace} containers.
```

Comment:
What does `determine` mean in practice? When I have to do this as a user, what do I do or what are my options when doing this?

Reply:
It can involve the user creating a new PVC, or finding an existing one (ex `kubectl get pvc -n mynamespace`) that the user wants to mount.

Reply:
Looking ahead onto the next step, the user needs to get the name of the PVC they want to mount.

Reply:
I would say that this step is a prerequisite: "Create a new PVC or determine an existing one in your user {orch-namespace} to mount to all workspaces containers."